src/main/resources/swagger/agora.yaml

```diff
@@ -948,10 +1013,10 @@ definitions:
         type: string
         description: Payload of method -- must be in WDL format
         default: |
-          task wc { command { cat ${File in_file} | wc -l } output { Int count = read_int(stdout()) }}
+          task wc {File in_file command { cat ${in_file} | wc -l } output { Int count = read_int(stdout()) }}
       entityType:
         type: string
-        description: Tyoe of the AgoraEntity -- Task or Workflow.
+        description: Type of the AgoraEntity -- Task or Workflow.
         default: Task
   AccessControl:
```

Comment:
For the description for roles, can we list the acceptable values? (Read, Write, Manage, Redact, All)